src/AltinnCore/Common/Services/Interfaces/IRepository.cs

```diff
@@ -463,9 +463,10 @@ public interface IRepository : IServicePackageRepository
         /// <summary>
         /// creates application  metadata for attachment
         /// </summary>
-        /// <param name="org">the organisation that owns the application</param>
-        /// <param name="appName">the application name</param>
-        void CreateApplication(string org, string appName);
+        /// <param name="org">The organisation code for the application owner</param>
+        /// <param name="app">the application name, e.g. "app-name-with-spaces"</param>
+        /// <param name="appTitleNb">The application title, e.g. "app name with spaces", at the moment it get stored as Norwegian title ("nb")</param>
```

Comment:
just appTitle

Reply:
David suggested to use appTitleNb so we are aware it get stored as "nb" appTitle since the code actually only do this. When this has been fixed to be stored as other languages on creation, we will change this variable name. The "nb" parameter had many other dependencies in codebase.

Reply:
Same as over.